gips/data/core.py

```diff
@@ -859,7 +859,9 @@ def asset_filenames(self, product):
         assets = self._products[product]['assets']
         filenames = []
         for asset in assets:
-            filenames.extend(self.assets[asset].datafiles())
+            emsg = 'no "{}" assets here'.format(asset)
+            with utils.error_handler(emsg, continuable=True):
+                filenames.extend(self.assets[asset].datafiles())
```

Comment:
I couldn't find any calls, except in `open_assets` which I can't find calls for either.  Did I miss them?  Change itself seems ok.

Reply:
I think I was using `open_assets` in IPython, which gave a less-than-indicative error message before.